app/frontend/src/components/ContributorForm/ContributorForm.test.tsx

```diff
@@ -47,63 +50,59 @@ describe("contributor form", () => {
 
   it("can be submitted filled", async () => {
```

Comment:
I am 90% sure this test is causing the issue but I don't know why @darrenvong

Reply:
The main difference I noticed is in this component, we are putting the `pb` stuff directly in the component where that stuff usually lives in `service` which gets mocked completely in tests, so I wonder if that's screwed up the mocking setup in some way...

Reply:
You're right, its a pb, I refactored it but same issue.
I am skipping it now - spent way too long on this. It's tested anyway in `Signup.test.tsx`